coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,38 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.filters.LanguageFilter import LanguageFilter
+from coalib.parsing.filters.CanDetectFilter import CanDetectFilter
+from coalib.parsing.filters.CanFixFilter import CanFixFilter
+
+
+class FilterHelper:
+    all_filters = {'language': LanguageFilter,
+                   'can_detect': CanDetectFilter,
+                   'can_fix': CanFixFilter}
+
+    @classmethod
+    def get_all_filters_str(cls, sep=', '):
+        return sep.join(sorted(cls.all_filters))
```

Comment:
Hmm. Okay that makes sense.

Although I imagine when I would now use your Filter stuff, I actually wouldn't look into your class when I need to build some help strings ;) Especially I wouldn't understand what they are exactly producing as output and where they are especially useful for, and then I'm usually faster writing some quick expression myself (thank you Python :D), using `all_filters`.